docs/Community/governance/growth-plan.md

```diff
@@ -0,0 +1,51 @@
+# Growth Plan
+
+## Introduction
+
+The Open Enclave SDK is an
+[Incubation stage](https://github.com/confidential-computing/governance/blob/master/project-progression-policy.md#incubation-stage)
+project of the Confidential Computing Consortium (CCC), with an approved
+[Technical Charter](/TECHNICAL_CHARTER.md).
+
+One requirement for Incubation stage projects is to develop, in
+conjunction with the mentor assigned by the CCC Technical Advisory
+Council, a growth plan for reaching the
+[Graduation stage](https://github.com/confidential-computing/governance/blob/master/project-progression-policy.md#graduation-stage)
+over time.
+
+## Graduation Stage Acceptance Criteria
+
+This section tracks the progress towards the Acceptance Criteria for the
+Graduation Stage, as defined by the CCC:
+
+- [ ] Have a defined [governing body](README.md) of at least 5 or more members
+(owners and core maintainers, or similar technical role), of which no
+more than 1/3 is affiliated with the same employer. In the case there
+are 5 governing members, 2 may be from the same employer.
+
+- [ ] Have a healthy number of [committers](../Committers.md#list-of-committers) from at least two organizations.
+A committer is defined as someone with the commit bit; i.e., someone who
```

Comment:
"someone with the commit bit" could be  more specific. does this refer to a GitHub permission, a Bors permission, or something else?

Reply:
To me it is clear enough... someone with permission to insert code into the master branch.   The wording is tooling independent.   (There are other things that I think are too subjective, but this isn't one I'm personally worried about.)